com.moddb.TotalChaosRetro.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.moddb.TotalChaosRetro</id>
+  <launchable type="desktop-id">com.moddb.TotalChaosRetro.desktop</launchable>
+  <name>Total Chaos Retro</name>
```

Comment:
I would personally use the full name in AppData:
```suggestion
  <name>Total Chaos Retro Edition</name>
```
Desktop file name can be just "Total Chaos Retro".

Reply:
But this is up to you. Both variants are correct.